src/index.js

```diff
@@ -61,239 +61,167 @@ class Radar {
 
   static getLocation(callback) {
     if (!callback) {
-      throw new Error(STATUS.ERROR_PARAMETERS);
+      throw new Error(STATUS.ERROR_MISSING_CALLBACK);
     }
 
-    Navigator.getCurrentPosition((status, { latitude, longitude, accuracy }) => {
-      if (status !== STATUS.SUCCESS) {
-        callback(status);
-        return;
-      }
-
-      callback(status, { latitude, longitude, accuracy });
-      return;
-    });
+    Navigator.getCurrentPosition()
+      .then((location) => {
+        callback(STATUS.SUCCESS, location);
```

Comment:
if we invoke as `callback(null, location)` and likewise throughout this file, I think our top-level functions can be passed to `util.promisify`